src/hotspot/share/utilities/globalDefinitions_xlc.hpp

```diff
@@ -1,6 +1,7 @@
 /*
  * Copyright (c) 1998, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
You need to update the year for the Oracle copyright. Adding additional copyright lines, such as IBM, is usually only done when you substantially change a file, i.e. add larger non-trivial portions of code.

Reply:
Ok. Thanks will keep that in mind. Updated the copyrights year.